pkg/controller/embercsi/embercsi_controller.go

```diff
@@ -309,6 +309,12 @@ func getEnvVars(ecsi *embercsiv1alpha1.EmberCSI, driverMode string) []corev1.Env
                         Value: ecsi.Spec.Config.EnvVars.X_CSI_PERSISTENCE_CONFIG,
 			},
 		)
+	} else { // Use CRD as the default persistence
+		envVars = append(envVars, corev1.EnvVar{
+                        Name: "X_CSI_PERSISTENCE_CONFIG",
+                        Value: fmt.Sprintf("{\"storage\":\"crd\",\"namespace\":%s}", ecsi.Namespace),
+			},
+		)
```

Comment:
Should we deploy Ember pods in the same namespace as the one we use for the CRD objects?

Reply:
I think at the moment only the CRD objects live in the 'default' namespace. Every other object that the operator creates lives in the same name space,. So the short answer is yes.

Reply:
The rest of the pods already get deployed to ember-csi. So short answer Yes.